content/blog/2019/11/2019-11-17-gsoc-report.adoc

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: "Google Summer of Code 2019 Report"
+tags:
+- gsoc
+- gsoc2019
+- jenkins
+- community
+- events
+author: martinda
+---
+
+GSoC is officially completed and we are happy to announce that five projects were successfully completed.
+We held the final presentations in late August and Google published the results on Sept 3rd.
+
+We would like to thank all our students, their mentors and all other contributors who proposed project ideas, participated in student selection, in community bonding and in further reviews.
+Google Summer of Code is a major effort which would not be possible without the active participation of the Jenkins community.
+
+The final presentations can be found here:
+
+* link:https://www.youtube.com/watch?v=g19o24uzy6c[Part 1 presentations]
+* link:https://www.youtube.com/watch?v=cMSbdGwGWp0[Part 2 presentations]
+
+== Summary
+
+
+This year GSoC was a year-round effort for the organization and some community members.
+I would like to start this report by thanking all participants, students, mentors, subject matter experts and community members who support the project and the students.
+We could not do this without you.
+We presented the
+link:https://docs.google.com/presentation/d/e/2PACX-1vQf5RLIJXq8cPEKbCIaA4Bn3tMy5ITB4z2oGJKwLOJz_ne2AiNuSSvXi51nMDDDgA/pub?start=false&loop=false&delayms=3000[2019 Jenkins GSoC report]
+at link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World San Francisco] conference.
+
+For the 2019 edition of GSoC, we started gathering project ideas in the last months of 2018, long before the official start of the program.
+We upgraded the GSoC project section of the website with lots of information for link:https://jenkins.io/projects/gsoc/[students and mentors].
+Students started to reach out very early on as well, many weeks before GSoC officially started.
+This was very motivating.
+Some students even started to work on project ideas before the official start of the program.
+
+This year the org admin team had some very difficult decisions to make.
+With lots of students, lots of projects and lots of mentors, we had to request the right number of slots and try to match the projects with the most chances of success.
```

Comment:
Is possible to elaborate on the significance of this number? What happens if it's too few, or too many?

Reply:
Never mind, there's some discussion about this below